### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by StrangerSon713** [team-abnormals/abnormals-core#114]
I was looking at uses for pumpkin slice, clicked arrow in jei to see more recipes, it crashed.
Using 3.1.1 due to Create crash.

---- Minecraft Crash Report ----
// This is a token for 1 free hug. Redeem at your nearest Mojangsta: [~~HUG~~]

Time: 6/16/21 10:29 AM
Description: Exception while ticking

java.lang.IllegalStateException: Tag endergetic:ender_fire_base_blocks used before it was bound
	at net.minecraft.tags.TagRegistry$NamedTag.func_232944_c_(TagRegistry.java:131) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.tags.TagRegistry$NamedTag.func_230235_a_(TagRegistry.java:142) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.block.Block.func_203417_a(Block.java:115) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:autumnity.mixins.json:BlockMixin,pl:mixin:APP:performant.mixins.json:block.BlockMixin,pl:mixin:APP:performant.mixins.json:forge.BlockMixin,pl:mixin:A}
	at com.minecraftabnormals.endergetic.common.blocks.EnderFireBlock.isEnderFireBase(EnderFireBlock.java:29) ~[?:3.0.0] {re:mixin,re:classloading}
	at net.minecraft.block.AbstractFireBlock.handler$zef000$enderFirePlacement(AbstractFireBlock.java:526) ~[?:?] {re:mixin,re:classloading,pl:mixin:APP:endergetic.mixins.json:AbstractFireBlockMixin,pl:mixin:A}
	at net.minecraft.block.AbstractFireBlock.func_235326_a_(AbstractFireBlock.java) ~[?:?] {re:mixin,re:classloading,pl:mixin:APP:endergetic.mixins.json:AbstractFireBlockMixin,pl:mixin:A}
	at net.minecraft.block.FireBlock.func_235494_a_(FireBlock.java:223) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.block.FireBlock.tryCatchFire(FireBlock.java:212) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.block.FireBlock.func_225534_a_(FireBlock.java:154) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.block.AbstractBlock$AbstractBlockState.func_227033_a_(AbstractBlock.java:620) ~[?:?] {re:mixin,re:classloading}
	at net.minecraft.world.server.ServerWorld.func_205338_b(ServerWorld.java:591) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:performant.mixins.json:world.ServerWorldBlockUpdateMixin,pl:mixin:APP:abnormals_core.mixins.json:ServerWorldMixin,pl:mixin:APP:endergetic.mixins.json:ServerWorldMixin,pl:mixin:APP:quark.mixins.json:ServerWorldMixin,pl:mixin:APP:cavesandcliffs.mixins.json:common.world.gen.ServerWorldMixin,pl:mixin:APP:performant.mixins.json:world.ServerWorldMixin,pl:mixin:APP:performant.mixins.json:entity.MobServerWorldMixin,pl:mixin:A}
	at net.minecraft.world.server.ServerTickList.func_205365_a(SourceFile:84) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:performant.mixins.json:world.ServerTickListMixin,pl:mixin:A}
	at net.minecraft.world.server.ServerWorld.func_72835_b(ServerWorld.java:336) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:performant.mixins.json:world.ServerWorldBlockUpdateMixin,pl:mixin:APP:abnormals_core.mixins.json:ServerWorldMixin,pl:mixin:APP:endergetic.mixins.json:ServerWorldMixin,pl:mixin:APP:quark.mixins.json:ServerWorldMixin,pl:mixin:APP:cavesandcliffs.mixins.json:common.world.gen.ServerWorldMixin,pl:mixin:APP:performant.mixins.json:world.ServerWorldMixin,pl:mixin:APP:performant.mixins.json:entity.MobServerWorldMixin,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:851) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:787) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:78) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:642) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.func_240783_a_(MinecraftServer.java:232) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_74] {}

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
	at net.minecraft.tags.TagRegistry$NamedTag.func_232944_c_(TagRegistry.java:131) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.tags.TagRegistry$NamedTag.func_230235_a_(TagRegistry.java:142) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.block.Block.func_203417_a(Block.java:115) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:autumnity.mixins.json:BlockMixin,pl:mixin:APP:performant.mixins.json:block.BlockMixin,pl:mixin:APP:performant.mixins.json:forge.BlockMixin,pl:mixin:A}
	at com.minecraftabnormals.endergetic.common.blocks.EnderFireBlock.isEnderFireBase(EnderFireBlock.java:29) ~[?:3.0.0] {re:mixin,re:classloading}
	at net.minecraft.block.AbstractFireBlock.handler$zef000$enderFirePlacement(AbstractFireBlock.java:526) ~[?:?] {re:mixin,re:classloading,pl:mixin:APP:endergetic.mixins.json:AbstractFireBlockMixin,pl:mixin:A}
	at net.minecraft.block.AbstractFireBlock.func_235326_a_(AbstractFireBlock.java) ~[?:?] {re:mixin,re:classloading,pl:mixin:APP:endergetic.mixins.json:AbstractFireBlockMixin,pl:mixin:A}
	at net.minecraft.block.FireBlock.func_235494_a_(FireBlock.java:223) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.block.FireBlock.tryCatchFire(FireBlock.java:212) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.block.FireBlock.func_225534_a_(FireBlock.java:154) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.block.AbstractBlock$AbstractBlockState.func_227033_a_(AbstractBlock.java:620) ~[?:?] {re:mixin,re:classloading}
	at net.minecraft.world.server.ServerWorld.func_205338_b(ServerWorld.java:591) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:performant.mixins.json:world.ServerWorldBlockUpdateMixin,pl:mixin:APP:abnormals_core.mixins.json:ServerWorldMixin,pl:mixin:APP:endergetic.mixins.json:ServerWorldMixin,pl:mixin:APP:quark.mixins.json:ServerWorldMixin,pl:mixin:APP:cavesandcliffs.mixins.json:common.world.gen.ServerWorldMixin,pl:mixin:APP:performant.mixins.json:world.ServerWorldMixin,pl:mixin:APP:performant.mixins.json:entity.MobServerWorldMixin,pl:mixin:A}
-- Block being ticked --
Details:
	Block location: World: (-248,38,-229), Chunk: (at 8,2,11 in -16,-15; contains blocks -256,0,-240 to -241,255,-225), Region: (-1,-1; contains chunks -32,-32 to -1,-1, blocks -512,0,-512 to -1,255,-1)
Stacktrace:
	at net.minecraft.world.server.ServerTickList.func_205365_a(SourceFile:84) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:performant.mixins.json:world.ServerTickListMixin,pl:mixin:A}
	at net.minecraft.world.server.ServerWorld.func_72835_b(ServerWorld.java:336) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:performant.mixins.json:world.ServerWorldBlockUpdateMixin,pl:mixin:APP:abnormals_core.mixins.json:ServerWorldMixin,pl:mixin:APP:endergetic.mixins.json:ServerWorldMixin,pl:mixin:APP:quark.mixins.json:ServerWorldMixin,pl:mixin:APP:cavesandcliffs.mixins.json:common.world.gen.ServerWorldMixin,pl:mixin:APP:performant.mixins.json:world.ServerWorldMixin,pl:mixin:APP:performant.mixins.json:entity.MobServerWorldMixin,pl:mixin:A}

-- Affected level --
Details:
	All players: 1 total; [ServerPlayerEntity['StrangerSon'/239, l='ServerLevel[New World]', x=-226.17, y=67.06, z=-188.85]]
	Chunk stats: ServerChunkCache: 2299
	Level dimension: minecraft:overworld
	Level spawn location: World: (-192,68,-176), Chunk: (at 0,4,0 in -12,-11; contains blocks -192,0,-176 to -177,255,-161), Region: (-1,-1; contains chunks -32,-32 to -1,-1, blocks -512,0,-512 to -1,255,-1)
	Level time: 37971 game time, 48437 day time
	Level name: New World
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
	Level weather: Rain time: 82104 (now: false), thunder time: 51619 (now: false)
	Known server brands: forge
	Level was modded: true
	Level storage version: 0x04ABD - Anvil
Stacktrace:
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:851) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:787) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:78) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:642) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.func_240783_a_(MinecraftServer.java:232) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_74] {}

-- System Details --
Details:
	Minecraft Version: 1.16.5
	Minecraft Version ID: 1.16.5
	Operating System: Mac OS X (x86_64) version 10.14.6
	Java Version: 1.8.0_74, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 783785880 bytes (747 MB) / 3234856960 bytes (3085 MB) up to 4056416256 bytes (3868 MB)
	CPUs: 8
	JVM Flags: 4 total; -Xss1M -Xmx4352m -Xms256m -XX:PermSize=256m
	ModLauncher: 8.0.9+86+master.3cf110c
	ModLauncher launch target: fmlclient
	ModLauncher naming: srg
	ModLauncher services: 
		/mixin-0.8.2.jar mixin PLUGINSERVICE 
		/eventbus-4.0.0.jar eventbus PLUGINSERVICE 
		/forge-1.16.5-36.1.0.jar object_holder_definalize PLUGINSERVICE 
		/forge-1.16.5-36.1.0.jar runtime_enum_extender PLUGINSERVICE 
		/accesstransformers-3.0.1.jar accesstransformer PLUGINSERVICE 
		/forge-1.16.5-36.1.0.jar capability_inject_definalize PLUGINSERVICE 
		/forge-1.16.5-36.1.0.jar runtimedistcleaner PLUGINSERVICE 
		/mixin-0.8.2.jar mixin TRANSFORMATIONSERVICE 
		/forge-1.16.5-36.1.0.jar fml TRANSFORMATIONSERVICE 
	FML: 36.1
	Forge: net.minecraftforge:36.1.0
	FML Language Providers: 
		javafml@36.1
		minecraft@1
	Mod List: 
		QuarkOddities-1.16.3.jar                          |Quark Oddities                |quarkoddities                 |1.16.3              |DONE      |NOSIGNATURE
		infernal-expansion-forge-1.3.jar                  |Infernal Expansion            |infernalexp                   |Release 1.3         |DONE      |NOSIGNATURE
		jei-1.16.5-7.7.0.106.jar                          |Just Enough Items             |jei                           |7.7.0.106           |DONE      |NOSIGNATURE
		abnormals_core-1.16.5-3.1.1.jar                   |Abnormals Core                |abnormals_core                |3.1.1               |DONE      |NOSIGNATURE
		buzzier_bees-1.16.5-3.0.1.jar                     |Buzzier Bees                  |buzzier_bees                  |3.0.1               |DONE      |NOSIGNATURE
		berry_good-1.16.5-4.1.1.jar                       |Berry Good                    |berry_good                    |4.1.1               |DONE      |NOSIGNATURE
		Fruitful-1.16.5-2.1.0.jar                         |Fruitful                      |fruitful                      |1.16.5-2.1.0        |DONE      |NOSIGNATURE
		cookielicious-1.16.5-1.2.2.jar                    |Cookielicious                 |cookielicious                 |1.2.2               |DONE      |NOSIGNATURE
		Enhanced-Mushrooms-1.16.5-3.0.8.jar               |Enhanced Mushrooms            |enhanced_mushrooms            |1.16.5-3.0.8        |DONE      |NOSIGNATURE
		caelus-forge-1.16.5-2.1.3.0.jar                   |Caelus API                    |caelus                        |1.16.5-2.1.3.0      |DONE      |NOSIGNATURE
		Better-Badlands-1.16.5-2.0.3.jar                  |Better Badlands               |better_badlands               |1.16.5-2.0.3        |DONE      |NOSIGNATURE
		extraboats-1.16.5-2.1.0.jar                       |Extra Boats                   |extraboats                    |2.1.0               |DONE      |NOSIGNATURE
		create-mc1.16.5_v0.3.1c.jar                       |Create                        |create                        |mc1.16.5_v0.3.1c+151|DONE      |NOSIGNATURE
		BetterCaves-Forge-1.16.4-1.1.2.jar                |YUNG's Better Caves           |bettercaves                   |1.16.4-1.1.2        |DONE      |NOSIGNATURE
		shutupexperimentalsettings-1.0.3.jar              |Shutup Experimental Settings! |shutupexperimentalsettings    |1.0.3               |DONE      |NOSIGNATURE
		mgui-1.16.5-3.2.0.jar                             |mgui                          |mgui                          |3.2.0               |DONE      |NOSIGNATURE
		Placebo-1.16.4-4.5.0.jar                          |Placebo                       |placebo                       |4.5.0               |DONE      |NOSIGNATURE
		comforts-forge-1.16.5-4.0.0.4.jar                 |Comforts                      |comforts                      |1.16.5-4.0.0.4      |DONE      |NOSIGNATURE
		citadel-1.7.1-1.16.5.jar                          |Citadel                       |citadel                       |1.7.1               |DONE      |NOSIGNATURE
		alexsmobs-1.10.1.jar                              |Alex's Mobs                   |alexsmobs                     |1.10.1              |DONE      |NOSIGNATURE
		Artifacts-1.16.5-2.9.3.jar                        |Artifacts                     |artifacts                     |1.16.5-2.9.3        |DONE      |NOSIGNATURE
		YungsApi-1.16.4-Forge-6.jar                       |YUNG's API                    |yungsapi                      |1.16.4-Forge-6      |DONE      |NOSIGNATURE
		OuterEnd-0.2.12.jar                               |The Outer End                 |outer_end                     |0.2.9               |DONE      |NOSIGNATURE
		environmental-1.16.5-1.0.0.jar                    |Environmental                 |environmental                 |1.0.0               |DONE      |NOSIGNATURE
		Bayou-Blues-1.16.5-1.0.5.jar                      |Bayou Blues                   |bayou_blues                   |1.16.5-1.0.5        |DONE      |NOSIGNATURE
		curioofundying-forge-1.16.5-5.2.0.0.jar           |Curio of Undying              |curioofundying                |1.16.5-5.2.0.0      |DONE      |NOSIGNATURE
		iceandfire-2.1.7-1.16.5.jar                       |Ice and Fire                  |iceandfire                    |2.1.7-1.16.5        |DONE      |NOSIGNATURE
		allurement-1.16.5-1.2.0.jar                       |Allurement                    |allurement                    |1.2.0               |DONE      |NOSIGNATURE
		forge-1.16.5-36.1.0-universal.jar                 |Forge                         |forge                         |36.1.0              |DONE      |22:af:21:d8:19:82:7f:93:94:fe:2b:ac:b7:e4:41:57:68:39:87:b1:a7:5c:c6:44:f9:25:74:21:14:f5:0d:90
		neapolitan-1.16.5-2.1.0.jar                       |Neapolitan                    |neapolitan                    |2.1.0               |DONE      |NOSIGNATURE
		antiqueatlas-5.4.4-forge.jar                      |Antique Atlas                 |antiqueatlas                  |5.4.4-forge         |DONE      |NOSIGNATURE
		forge-1.16.5-36.1.0-client.jar                    |Minecraft                     |minecraft                     |1.16.5              |DONE      |NOSIGNATURE
		upgrade_aquatic-1.16.5-3.1.0.jar                  |Upgrade Aquatic               |upgrade_aquatic               |3.1.0               |DONE      |NOSIGNATURE
		cofh_core-1.16.4-1.2.1.jar                        |CoFH Core                     |cofh_core                     |1.2.1               |DONE      |NOSIGNATURE
		endergetic-1.16.4-3.0.0.jar                       |The Endergetic Expansion      |endergetic                    |3.0.0               |DONE      |NOSIGNATURE
		FarmersDelight-1.16.5-0.4.3.jar                   |Farmer's Delight              |farmersdelight                |1.16.5-0.4.3        |DONE      |NOSIGNATURE
		repurposed_structures-1.16.5-2.7.5-forge.jar      |Repurposed Structures         |repurposed_structures         |1.16.5-2.7.5-forge  |DONE      |NOSIGNATURE
		theoneprobe-1.16-3.1.4.jar                        |The One Probe                 |theoneprobe                   |1.16-3.1.4          |DONE      |NOSIGNATURE
		Abundance-1.16.5-1.0.5.jar                        |Abundance                     |abundance                     |1.16.5-1.0.5        |DONE      |NOSIGNATURE
		oreganized-1.16.4-0.4.4-beta.jar                  |Oreganized Mod                |oreganized                    |1.16.4-0.4.4-beta   |DONE      |NOSIGNATURE
		archers_paradox-1.16.4-1.2.0.jar                  |Archer's Paradox              |archers_paradox               |1.2.0               |DONE      |NOSIGNATURE
		CavesAndCliffsBackport-1.16.5-5.2.jar             |Caves & Cliffs Backport       |cavesandcliffs                |5.2                 |DONE      |NOSIGNATURE
		DarkerDepths-1.16.5-1.0.1.jar                     |Darker Depths                 |darkerdepths                  |1.0.1}              |DONE      |NOSIGNATURE
		personality-1.16.5-1.0.2.jar                      |Personality                   |personality                   |1.0.2               |DONE      |NOSIGNATURE
		atmospheric-1.16.5-3.1.0.jar                      |Atmospheric                   |atmospheric                   |3.1.0               |DONE      |NOSIGNATURE
		curios-forge-1.16.5-4.0.5.2.jar                   |Curios API                    |curios                        |1.16.5-4.0.5.2      |DONE      |NOSIGNATURE
		tetra-1.16.5-3.11.1.jar                           |Tetra                         |tetra                         |3.11.1              |DONE      |NOSIGNATURE
		tetranomicon-1.3.jar                              |Tetranomicon                  |tetranomicon                  |1.3                 |DONE      |NOSIGNATURE
		savageandravage-1.16.5-3.1.0.jar                  |Savage & Ravage               |savageandravage               |3.1.0               |DONE      |NOSIGNATURE
		FastWorkbench-1.16.4-4.5.1.jar                    |FastWorkbench                 |fastbench                     |4.5.1               |DONE      |NOSIGNATURE
		autumnity-1.16.5-2.1.1.jar                        |Autumnity                     |autumnity                     |2.1.1               |DONE      |NOSIGNATURE
		AutoRegLib-1.6-49.jar                             |AutoRegLib                    |autoreglib                    |1.6-49              |DONE      |NOSIGNATURE
		Quark-r2.4-312.jar                                |Quark                         |quark                         |r2.4-312            |DONE      |NOSIGNATURE
		abnormals_delight-1.16.5-1.1.1.jar                |Abnormals Delight             |abnormals_delight             |1.1.1               |DONE      |NOSIGNATURE
		YungsExtras-Forge-1.16.4-1.0.jar                  |YUNG's Extras                 |yungsextras                   |Forge-1.16.4-1.0    |DONE      |NOSIGNATURE
		performant-1.16.2-5-3.66m.jar                     |Performant                    |performant                    |3.56m               |DONE      |NOSIGNATURE
		Peculiars-1.16.4-1.1.0.jar                        |Peculiars                     |peculiars                     |1.1.0               |DONE      |NOSIGNATURE
		BetterStrongholds-1.16.4-1.1.jar                  |YUNG's Better Strongholds     |betterstrongholds             |1.16.4-1.1          |DONE      |NOSIGNATURE
		structurize-0.13.177-ALPHA-universal.jar          |Structurize                   |structurize                   |0.13.177-ALPHA      |DONE      |NOSIGNATURE
		minecolonies-0.14.221-BETA-universal.jar          |MineColonies                  |minecolonies                  |0.14.221-BETA       |DONE      |NOSIGNATURE
		FastFurnace-1.16.4-4.4.0.jar                      |FastFurnace                   |fastfurnace                   |4.4.0               |DONE      |NOSIGNATURE
		sapience-1.16.4-1.1.1.jar                         |Sapience                      |sapience                      |1.1.1               |DONE      |NOSIGNATURE
		cobbler-1.6.1.jar                                 |Shulkers Faithful Factories   |cobbler                       |1.6.1               |DONE      |NOSIGNATURE
		AppleSkin-mc1.16.2-forge-1.0.14.jar               |AppleSkin                     |appleskin                     |1.0.14              |DONE      |NOSIGNATURE
		curiouselytra-forge-1.16.5-4.0.2.3.jar            |Curious Elytra                |curiouselytra                 |1.16.5-4.0.2.3      |DONE      |NOSIGNATURE
		AI-Improvements-1.16.2-0.3.0.jar                  |AI-Improvements               |aiimprovements                |0.3.0               |DONE      |NOSIGNATURE
		bgcore-1.16.5-3.2.jar                             |bgcore                        |bgcore                        |3.2                 |DONE      |NOSIGNATURE
		Orcinus-Caves-1.0.0.jar                           |Orcinus's Cavern              |cavern                        |1.0-SNAPSHOT        |DONE      |NOSIGNATURE
		FastLeafDecay-v25.jar                             |FastLeafDecay                 |fastleafdecay                 |v25                 |DONE      |NOSIGNATURE
		expandability-2.0.1-forge.jar                     |ExpandAbility                 |expandability                 |2.0.1               |DONE      |NOSIGNATURE
		BetterMineshafts-Forge-1.16.4-2.0.4.jar           |YUNG's Better Mineshafts      |bettermineshafts              |1.16.4-2.0.4        |DONE      |NOSIGNATURE
	Crash Report UUID: a825dd7b-15f5-4621-a20f-d090ac9245db
	Player Count: 1 / 8; [ServerPlayerEntity['StrangerSon'/239, l='ServerLevel[New World]', x=-226.17, y=67.06, z=-188.85]]
	Data Packs: mod:cavern, mod:bettermineshafts, vanilla, mod:quarkoddities (incompatible), mod:infernalexp (incompatible), mod:abnormals_core, mod:buzzier_bees, mod:berry_good, mod:fruitful (incompatible), mod:cookielicious, mod:enhanced_mushrooms, mod:caelus, mod:better_badlands, mod:extraboats, mod:bettercaves (incompatible), mod:shutupexperimentalsettings (incompatible), mod:mgui (incompatible), mod:placebo (incompatible), mod:comforts, mod:citadel (incompatible), mod:alexsmobs, mod:artifacts, mod:yungsapi, mod:outer_end, mod:environmental, mod:bayou_blues, mod:curioofundying, mod:iceandfire (incompatible), mod:allurement, mod:forge, mod:neapolitan, mod:antiqueatlas, mod:upgrade_aquatic, mod:cofh_core (incompatible), mod:endergetic, mod:farmersdelight, mod:repurposed_structures, mod:theoneprobe, mod:abundance, mod:archers_paradox (incompatible), mod:cavesandcliffs (incompatible), mod:darkerdepths, mod:personality, mod:atmospheric, mod:curios, mod:tetra, mod:tetranomicon, mod:savageandravage, mod:fastbench (incompatible), mod:autumnity, mod:autoreglib (incompatible), mod:quark (incompatible), mod:abnormals_delight, mod:yungsextras, mod:performant (incompatible), mod:peculiars, mod:betterstrongholds, mod:structurize, mod:minecolonies (incompatible), mod:fastfurnace (incompatible), mod:sapience, mod:appleskin, mod:curiouselytra, mod:aiimprovements, mod:bgcore, mod:fastleafdecay (incompatible), mod:expandability, mod:jei, mod:create, mod:oreganized, mod:cobbler
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'forge'

**Comment by SmellyModder** [comment]
Not related to Abnormals Core.
See https://github.com/BlueDuckYT/The-Outer-End/issues/47

**Comment by Brun333rp** [comment]
that crash was probably my fault. some while ago, I had uploaded a translation for the Outer End, and alongside the translation I uploaded files containing recipes for Environmental sawmill recipes, tags for Endergetic's ender fire base blocks and recipes for Farmer's Delight cutting board recipes. it ended that the sawmill recipes didn't worked properly and the game crashed when loading Endergetic and Outer End together. that was my fault, I'm sorry everyone for ruining your gameplay. I should have just uploaded the translation and left the rest as it was prior to that.
